docs/src/extend/plugins/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Community Plugins
+tags: [Documentation]
+---
+
+import { PluginCardsList } from '@site/src/components/PluginCardsList';
+
+The following pages contain a list of available community plugins. You can use these plugins directly or as a reference for creating your own plugins.
+
+## Plugins List
+
+The list can be filtered by keywords and/or text to find plugins which are useful to you.
+
+<PluginCardsList />
```

Comment:
It might be useful to add a link to the *Add or edit a plugin* section of the [`tedge-docs/README.md``](https://github.com/thin-edge/tedge-docs/blob/main/README.md).

Reply:
Resolved by 3465e791d6adfd1f45054d9cee708314b64e11af